test/core/base/audit.js

```diff
@@ -535,16 +535,19 @@ describe('Audit', function () {
 			var targetRule = mockRules[mockRules.length - 1],
```

Comment:
You're not testing that assets are preloaded

Reply:
I will be extracting all of the preload logic that needs to be in run into a separate PR. Will tackle this comment there.